### PR TITLE
Remove context data

### PR DESCRIPTION
**Pull request by PaulLeCam**

### CODE REVIEW

**Comment by howleysv** [comment]
We could move tz to the connection user agent and just generate this client data on the server if needed

**Comment by PaulLeCam** [comment]
For the user agent by top-level you mean even outside of `context`? If so how will the client provide it? Should we add an extra key to the `bot.post` payload?

**Comment by bismark** [comment]
@PaulLeCam yah thats my thinking... context would be purely generated by the server
@howleysv that would require us giving some sort of client identifier to the bot to look up that context with... not saying we can't do that, just pointing it out

**Comment by howleysv** [comment]
@bismark Well how do you expect the bot to get this data now?

**Comment by PaulLeCam** [comment]
From my understanding of Shane's comment, instead of providing the TZ here the client would provide it as part of the user agent when making the WS connection, no?
If we do this + making `bot.subscription.*` do the `/post` request, the client wouldn't need to handle the `context` at all anymore.

**Comment by howleysv** [comment]
@PaulLeCam Yeah this is the benefit I'm thinking about

**Comment by bismark** [comment]
@howleysv they would be able to use context info to fetch the data, e.g. conversation id to fetch the conversation metadata. there isn't an id for "calling client" however. my proposal is for the client to also not touch the context, but just to add a user agent top level field to the posts..

**Comment by howleysv** [comment]
So you're talking about enforcing the field then. We could do that on the server either and control the data that goes in there.

File: source/index.html.md [hunk 1]
```diff
@@ -251,16 +240,11 @@ POST https://bot-api.example.com/post HTTP/1.1
...
   "data": {
     "action": "new_post_form"
   },
   "context": {
-    "user": {
-      "id": "<unique user ID>",
-      "username": "alice",
-      "name": "Alice"
-    }
+    "user_id": "<unique user ID>"
```
Comment (howleysv) [inline source/index.html.md]: When would there be a `conversation_id` in here?

**Comment by howleysv** [comment]
We could use context everywhere?

**Comment by PaulLeCam** [comment]
Yeah I was wondering about the other requests, I think we can either document it as the context being in the root, or we could provide this data in `context` to make it similar to `post`

File: source/index.html.md (hunk 1 above)
Comment (PaulLeCam) [inline source/index.html.md]: 👍 updated

**Pull request merged**